changelog/fragments/1707143167-send-package-version-to-components.yaml

```diff
@@ -0,0 +1,34 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: Agent information is sent to components together with the connection information.
```

Comment:
Is this really informational for our users? I don't think they care, this is more an internal change and not something that affects the usage of the Elastic Agent.

Reply:
I tend to agree, this change is more an internal change than something the users should pay attention at

Reply:
+1 we can remove the changelog entry here.